packages/database/migrations/0001_create_object_item.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `object_item` (
+  `id` varchar(32) NOT NULL,
+  `object_id` text NOT NULL,
+  `object_list_id` int,
+  `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `created_by` varchar(50) NOT NULL,
+  CONSTRAINT `object_item_id` PRIMARY KEY(`id`),
+  CONSTRAINT `object_item_id_object_list_id_unique` UNIQUE(`id`,`object_list_id`)
+);
+--> statement-breakpoint
+CREATE INDEX `id` ON `object_item` (`id`);--> statement-breakpoint
```

Comment:
Mmm - I just realized this:

1. The `id` is not unique. But the name `id` could give that impression (since the `id` in `object_list` is unique). So the meaning of `id` in this column differs from the same column in the other table.
2. If we ever want to add a 'true' `id` column (i.e. similar to the `id` column in `object_list`) then the current column stands in the way - the name is already taken.
3. Would it then be better to rename it to e.g. `object_id` (reasoning: it's the ID that we've given to the object - it's not the ID of the `object_item` record)? And to rename `object_id` to `object_iri` (I don't like the `_iri` suffix, but I don't have an alternative)?

Sorry for the hassle!